docs/codelist-project.md

```diff
@@ -1,19 +1,37 @@
 ## Adding a codelist
-Take a look at the `codelists/codelists.txt` file in the repo, and note the structure of the existing example codelists that shipped with the research template:
+
+Your example research template doesn't include any codelists but the folder strucure and text files that are needed to include codelists already exist. 
+Take a look at the `codelists/codelist.txt` file in the repo, this file is currently empty but you can add one (or more) strings to this file that specify the codelists that need for your research project. 
```

Comment:
Could you change the path to `codelists/codelists.txt` here and below? (note the plural) And although it's a nitpick (and so you are free to ignore me!), might it be clearer to replace "strings" with "lines" both here and below? Finally, I think:

> ... the codelists that need for your research project.

should be:

> ... the codelists that you need for your research project.